src/main/resources/network/brightspots/rcv/hints_candidates.txt

```diff
@@ -4,6 +4,9 @@ Fill in the fields and click the Add button:
 
 Name (required): E.g.: Dave Harris.
 
-Code (optional): Some CVR files use codes in lieu of full candidate name. E.g.: "JCD" or "14".
+Codes or Aliases (optional): Enter one code or alias per line.
+Codes are used by some CVR files in lieu of full candidate name. E.g.: "JCD" or "14".
+Aliases are helpful when candidate names are not normalized across jurisdictions, e.g. "JAMES" may be an alias for "James".
```

Comment:
Do we really want these to be case-sensitive? Seems to me like it'd probably be best to run them all through `String.upper()` before processing them in the backend.

I think a better example might be "Jim" might be an alias for "James".